src/apiCalls.js

```diff
@@ -1,3 +1,3 @@
-export default function fetchData() {
-    return fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies').then(response => response.json()).catch(error => console.log(error))
+export default function fetchData(path) {
+    return fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/${path}`).then(response => response.json()).catch(error => console.log(error))
```

Comment:
Probably a good idea to move this catch into the app.js - so we can _do_ something with that error message that would benefit the user!